scripts/test-pixell

```diff
@@ -2,11 +2,31 @@
 
 import os,sys
 import unittest
+
+# Must do the thread setup before we import pixell tests, otherwise those
+# dependent on the threadpool will fail.
+
+import multiprocessing
+max_threads = multiprocessing.cpu_count()
+assert max_threads>=1
+
+if sys.platform == "darwin":
+    if "DUCC0_NUM_THREADS" not in os.environ:
+        os.environ["DUCC0_NUM_THREADS"] = str(max_threads)
```

Comment:
Similar comment as before for @amaurea and @mreineck , how is this envvar different from OMP_NUM_THREADS

Reply:
`ducc0` uses `pthreads` and its own threadpool, not OMP, so it has its own environment variables for setting the threadpool options.